.tekton/wait-for-image-task.yaml

```
@@ -30,9 +34,12 @@ spec:
 
       echo "Image $(params.IMAGE) found."
 
-      digest="$(skopeo inspect \
+      infos="$(skopeo inspect \
         --retry-times 10 \
-        --format '{{.Digest}}' \
+        --format '{{.Digest}} {{ index .Labels "vcs-ref" }} {{ index .Labels "source-location" }}' \
         --no-tags \
         "docker://$(params.IMAGE)")"
-      echo -n "$digest" | tee "$(results.IMAGE_DIGEST.path)"
+
+      echo -n $(cut -f1 -d' ' <<< "${infos}") | tee "$(results.IMAGE_DIGEST.path)"
+      echo -n $(cut -f2 -d' ' <<< "${infos}") | tee "$(results.GIT_REF.path)"
+      echo -n $(cut -f3 -d' ' <<< "${infos}") | tee "$(results.GIT_REPO.path)"
```

Please quote substitutions
```suggestion
      echo -n "$(cut -f1 -d' ' <<< "${infos}")" | tee "$(results.IMAGE_DIGEST.path)"
      echo -n "$(cut -f2 -d' ' <<< "${infos}")" | tee "$(results.GIT_REF.path)"
      echo -n "$(cut -f3 -d' ' <<< "${infos}")" | tee "$(results.GIT_REPO.path)"
```

---

Done in https://github.com/stackrox/stackrox/pull/13577/commits/709e0fa713ea1731877488fdaaebad6e032aebb5